includes/definitions.inc.php

```diff
@@ -1906,6 +1906,12 @@
 $config['os'][$os]['icon']             = 'enterasys';
 $config['os'][$os]['ifname']           = 1;
 
+ // ABB DPA UPScale (Generex CS121 SNMP Adapter)
+$os = 'abbdpaupscale';
```

Comment:
Is this the best OS name to use? Seems more descriptive of a single product rather than a firmware / os name.